source/usage-and-configuration/swagger/introduction.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Swagger
+---
+
+# Swagger
+
+The theme provides tools to read and parse swagger files and generate documentations out of it.
+
+## Tags
+It provides following two tags which can grab the swagger schema from local or a remote location and create beautiful and easy to read documents.
+* **[swagger_ui](./swagger-ui.html) -**  This tag generates [swagger ui](https://swagger.io/swagger-ui) from the swagger schema.
+* **[swagger_to_html](./swagger-to-html.html) -**  This tag generates a simple and clean user interface that contains all the useful information present in the swagger specification with examples.
+
+
+## Decorators
+The theme also provides features to decorate/modify the swagger spec before using it.
+A use case that can explain the rationale behind this feature is, sometime we might want to modify the swagger and remove some endpoints from being exposed in the documentation. 
+
+Decorators provided by theme :
+## Excluding Definitions
+You can add a `x-doc` object with `exclude: true` property to remove spec definitions/objects from being processed.
+The flag can be added in the following places.
+  * **Excluding path -** To remove a complete path from the spec add the `x-doc` object to that path.
+
+    For instance, to remove `/pets` from the following sample schema.
+    ```yaml
+    swagger: "2.0"
+    paths:
+      /pets:
+        x-doc:
+          exclude: true
+    ```
+  * **Excluding operation -** To remove an operation from the spec you can add `x-doc` object to that operation.
+
+    For instance, to remove `get` operaton for `/pet` path in the following sample schema.
+    ```yaml
+    swagger: "2.0"
+    paths:
+      /pets:
+        get:
+          x-doc:
+            exclude: true
+          description: "Returns all pets from the system that the user has access to"
+    ```
+  * **Excluding parameters -** To remove parameters, add `x-doc` object to the parameter. If you add `x-doc` to a parameter definition, the parameter will be skipped from every place where it is referred.
```

Comment:
add an `x-doc`. . . from every place where it is referenced

Reply:
👍